base_set/cards/gardens.py

```diff
@@ -0,0 +1,12 @@
+from core.card import KingdomCard
+from core.card import CardType
+
+
+class GardensCard(KingdomCard):
+    types = (CardType.VICTORY,)
+    base_cost = 4
+    
+    @classmethod
+    def victory_points(cls, player, game_state):
+        deck_size = len(game_state.get_deck(player))
+        return (deck_size // 10) * 1
```

Comment:
why `* 1`?

Reply:
eh card text sounded like this "Worth 1 for every 10 cards..." not needed though